packages/localizations/src/en-US.ts

```diff
@@ -458,6 +458,7 @@ export const enUS: LocalizationResource = {
       'Sign up unsuccessful due to failed security validations. Please refresh the page to try again or reach out to support for more assistance.',
     captcha_unavailable:
       'Sign up unsuccessful due to failed bot validation. Please refresh the page to try again or reach out to support for more assistance.',
+    external_account_missing_refresh_token: 'This account has been disconnected',
```

Comment:
A better message might be the following:
`This account is missing a refresh token` or `There is no associated refresh token with this account`

Maybe we need to provide the steps that the user needs to do in order to resolve this e.g. Try to reconnect your account or sign in

Reply:
I understand were you are coming from, but we are explicitly using a very simple message here, because this is something the end user will see. I am confident that unless you are a developer you are not aware of what a "refresh token" is, so the error message would not be that useful to you.